test/unit/stop_test.lua

```diff
@@ -0,0 +1,34 @@
+local expirationd = require("expirationd")
+local fiber = require("fiber")
+local t = require("luatest")
+local g = t.group("stop")
+
+local helpers = require("test.helper")
+
+g.before_all(function()
+    helpers.init_spaces(g)
+end)
+
+g.after_each(function()
+    helpers.truncate_spaces(g)
+end)
+
+function g.test_cancel_on_pcall()
+    local function on_full_scan_complete()
+        pcall(function() fiber.sleep(1) end)
+    end
+
+    local task = expirationd.start("clean_all", g.tree.id, helpers.is_expired_true, {
+        full_scan_delay = 123123123,
+        on_full_scan_complete = on_full_scan_complete
+    })
+    helpers.retrying({}, function()
+        t.assert(task.worker_fiber)
+    end)
+    -- we need to execute in a separate fiber,
+    -- since pcall does not check testcancel and stop may freeze up
```

Comment:
Sentence starts with uppercase and dot at the end.
https://www.tarantool.io/en/doc/latest/dev_guide/lua_style_guide/

Reply:
Sorry, but I could not find in the guideline which you attached the words uppercase and dot, perhaps we need to leave a ticket to the documentation. And we will probably need to create a ticket in `expd` to redo the old comments so that they also match the comments style-guide

Reply:
https://github.com/tarantool/doc/issues/2331